src/ResourceManager/ApiManagement/Commands.ApiManagement/help/Add-AzureRmApiManagementApiToProduct.md

```diff
@@ -37,7 +37,7 @@ This command adds the specified API to the specified product.
 Specifies the ID of an API to add to a product.
 
 ```yaml
-Type: System.String
```

Comment:
We need these parameter types to be full name types.  Please update the help using this:

Get-ChildItem <path to repo>/src/ResourceManager/ApiManagement/Commands.ApiManagement/help | Update-MarkdownHelp -AlphabeticParamsOrder -UseFullTypeName

I am working with platyPS to get this parameter into Update-MarkdownHelpModule, and will update the help generation doc as soon as that is release.  Thanks!